search.mixer

```diff
@@ -3,49 +3,22 @@
 <mixertitle>Search the <mixer var="GAP"/> web site</mixertitle>
 
 <hr/>
-<p>Check the boxes to search that area of the archive.</p>
-<p>You can see the sentence(s) the search words appear in by checking the
-<tt>Display Context</tt> box below.</p><p><b>Note</b>
-that displaying the context can give quite lengthy output for more common
-search words.</p>
-<form action="Search/search.cgi">
+<p>You can use Google to search within the complete GAP website, manuals of GAP and packages, and GAP Forum archives.</p>
 
-<pre>
-<input type="checkbox" name="FORUM" value="1" checked="checked"/> Gap Forum Archives
-<input type="checkbox" name="OLDFORUM" value="1"/> Pre-2004 Forum Archives
-<input type="checkbox" name="MAN4" value="1" checked="checked"/> Gap 4 Manual
-<input type="checkbox" name="MANUAL3" value="1"/> Gap 3 Manual
-<input type="checkbox" name="WEB" value="1"/> This Website
-</pre>
-<p>Enter Search Keyword: <input name="WORD"/></p>
-<pre><input type="checkbox" name="CONTEXT" value="1"/> Display context
-</pre>
-<p><input type="submit" value="Perform Search"/> <input type="reset"
-value="Clear Selection"/></p>
+<ul>
+<li><a href="https://www.google.co.uk/search?q=site%3Awww.gap-system.org%3A">Complete GAP website (including manuals)</a></li>
+<li><a href="https://www.google.co.uk/search?q=site%3Awww.gap-system.org/Manuals/%3A">Manuals of GAP and packages</a></li>
+<li><a href="https://www.google.co.uk/search?q=site%3Ahttp://mail.gap-system.org/pipermail/forum/%3A">GAP Forum Archives since December 2003</a></li>
+<li><a href="https://www.google.co.uk/search?q=site%3Awww.gap-system.org/ForumArchive/%3A">Pre-2004 Forum Archives</a></li>
+</ul>
 
-</form>
 <h3>Search Tips</h3>
 <ul>
-<li>If your search keyword has an accented letter, then just use the letter
-without the accent.</li>
-<li>The search will look for files with all your search words in unless you
-type
-<tt>or</tt> between them.</li>
-<li>You can look for prefixes by adding a '<tt>*</tt>' at the end. e.g.
-<tt>grou*</tt> will find <b>group, groups, groupring</b>, etc.</li>
-
-<li>When <tt>Display Context</tt> is checked, some of the sentences that
-appear
-may end with a link saying <b>Go directly to this paragraph</b>.
-These links will take you straight to the paragraph that the sentence is in.
-</li>
+<li>Click on one of the links above and type the search term after the URL in the search box</li>
+<li>To enforce the search of a particular command, you may need to enclose it in quotes
+(otherwise e.g. entering CharacterTable will also return results mentioning character tables)</li>
+<li>Please note that GAP Forum Archives since December 2003 are not included in the complete GAP website search</li>
```

Comment:
I would add here "so you need to explicitly click on "GAP Forum Archives since December 2003" to search there"